src/common/util/include/openvino/util/file_util.hpp

```diff
@@ -119,7 +119,7 @@ bool directory_exists(const std::string& path);
  * @param[in]  path  The file name
  * @return     file size
  */
-inline uint64_t file_size(const char* path) {
+inline int64_t file_size(const char* path) {
```

Comment:
There are two more `file_size` functions in this file which just wrap use of this function.
I think they should return `int64_t` also, e.g. to correct forward return value of `-1`.